File: Trabalho de programação/media_acima_da_media.c
#include <stdio.h>

int main() {
    int n, i;
    printf("Quantos numeros voce deseja inserir? ");
    scanf("%d", &n);

    int numeros[n];
    int soma = 0;

    for (i = 0; i < n; i++) {
        printf("Insira o numero %d: ", i + 1);
        scanf("%d", &numeros[i]);
        soma += numeros[i];
    }

    float media = (float)soma / n;
    printf("Media: %.2f\n", media);
    printf("Numeros acima da media:\n");

    for (i = 0; i < n; i++) {
        if (numeros[i] > media) {
            printf("%d ", numeros[i]);
        }
    }

    return 0;
}

File: Trabalho de programação/MediaAcimaDaMedia.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Quantos números você deseja inserir? ");
        int n = scanner.nextInt();

        int[] numeros = new int[n];
        int soma = 0;

        for (int i = 0; i < n; i++) {
            System.out.print("Insira o número " + (i + 1) + ": ");
            numeros[i] = scanner.nextInt();
            soma += numeros[i];
        }

        float media = (float)soma / n;
        System.out.printf("Média: %.2f\n", media);
        System.out.println("Números acima da média:");

        for (int i = 0; i < n; i++) {
            if (numeros[i] > media) {
                System.out.print(numeros[i] + " ");
            }
        }
    }
}
